packages/sockethub/src/middleware/validate.test.data.ts

```diff
@@ -260,5 +258,51 @@ export default [
       "context":"irc",
       "target":"irc://irc.dooder.net/a-room"
     }
+  },
+  {
+    "name":"missing @type property",
+    "valid":false,
+    "type":"message",
+    "input":{
+      "actor": { "@id": "irc://uuu@localhost", "@type": "person" },
+      "context":"irc",
+      "target": { "@id": "irc://irc.dooder.net/a-room", "@type": "room" }
+    },
+    "error": "Error: actvity-stream schema validation failed: " +
+      "must have required property \'@type\'"
+  },
+  {
+    "name":"invalid context property",
+    "valid":false,
+    "type":"message",
+    "input":{
+      "actor": { "@id": "irc://uuu@localhost", "@type": "person" },
+      "@type":"foo",
+      "context": "foobar",
+      "target": { "@id": "irc://irc.dooder.net/a-room", "@type": "room" }
+    },
+    "error": "Error: platform context foobar not registered with this sockethub instance."
+  },
+  {
+    "name":"missing actor property",
+    "valid":false,
+    "type":"message",
+    "input":{
+      "@type": "foo",
+      "context":"irc",
+      "target": { "@id": "irc://irc.dooder.net/a-room", "@type": "room" }
+    },
+    "error": "Error: actvity-stream schema validation failed: " +
+      "must have required property \'actor\'"
+  },
+  {
+    "name":"traditional message",
+    "valid":true,
+    "type":"message",
+    "input":{
+      "@type": "foo",
+      "context": "irc",
+      "actor": { "@id": "irc://uuu@localhost", "@type": "person" }
+    }
```

Comment:
If I see correctly, the tests for failing validations are also testing the actual error message. Then there might be a test missing for this error https://github.com/sockethub/sockethub/pull/573/files#diff-2e9c7a22f756a8dfb626c2786ace21077766c34452d8f439a2638916a8eeff2eR74

Reply:
Well, while it uses the functionality of platform.js it's technically not testing that platformLoad method. I am avoiding writing tests for the JS files in bootstrap. ;)